quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -170,6 +171,11 @@ public void parseAndRun(List<String> cliArgs) {
                 addCommandOptions(cliArgs, cl);
             }
 
+            result = null;
+            if (Optional.ofNullable(currentCommand).map(AbstractCommand::getName).filter(Start.NAME::equals).isPresent()) {
+                reduceMemoryForStart(cmd);
```

Comment:
What is the real impact of this optimization? I'd like to avoid hacky optimizations unless they have significant and real impact.

Reply:
Ok, removed and added comments explaining what could be done next.